bots/happinessncheer.bot.js

```diff
@@ -0,0 +1,48 @@
+const Slack = require('pico-slack');
+const cron = require('node-schedule');
+const {differenceInCalendarDays} = require('date-fns');
+
+const peeps = [
+	`Rbb`,
+	`Christian`,
+	`Evelyn`,
+	`LP`,
+	`Scott`,
+	`Rachel`,
+	`Tina`,
+	`David`,
+	`Katie`,
+	`Meg`,
+	`Greg`,
+	`Kellen`,
+	`Cathleen`,
+	`Chris`,
+	`Simon`,
+	`Carly`,
+	`Jared`,
+	`Mark`,
+	`Jenny`,
+	`Conrad`,
+	`Ross`,
+	`Ryan`,
+];
+
+const PeepOffset = 13; //So whatever date it is we land on the right person
+
+const getSuggester = (offset=0) =>{
+	const now = new Date();
+	const yearStart = new Date(now.getFullYear(), 0, 0);
+	const delta = differenceInCalendarDays(now, yearStart)
+	return peeps[(delta + offset + PeepOffset) % peeps.length];
```

Comment:
I don't quite understand how `PeepOffset` works!

Reply:
We started at a "random" spot in the stack of people, so without this static offset today would be katie's turn, but it's actually ryan's. This offset just moves everything up to be in line with what people are expecting. 

We can also increase/decrease this to tweak if we want to skip a day, to bump someone around.

Reply:
If we're going to use PeepOffset to handle skipping/delaying someone's turn, should we make it a config so it's easier to update on the fly?

Reply:
It's happened so infrequently that the difference between it being in configs versus code isn't that big of a deal. If the project was larger with a more complex deployment than sure. But changing this value versus changing a config is nearly the same effort.